lib/store/src/args.ts

```diff
@@ -101,7 +101,8 @@ export const validateOptions = (args: Args, argTypes: ArgTypes): Args => {
     const isValidArray = isArray && invalidIndex === -1;
 
     if (args[key] === undefined || options.includes(args[key]) || isValidArray) {
```

Comment:
This check for `args[key] === undefined` seems a little pointless if we later do `if (key in args)`. Do we understand why the code was there in the first place? We should get rid of it, and perhaps figure that out.

Reply:
Or does it remain here to avoid the warning lower down?